tools/code_format/glint.sh

```diff
@@ -20,7 +20,7 @@ check_mixed_tabs_spaces () {
     tabbed=$(grep -cP "^\t" "$1")
     spaced=$(grep -cP "^ " "$1")
     if [[ $tabbed -gt 0 ]] && [[ $spaced -gt 0 ]]; then
-        echo "mixed tabs and spaces: ${1}" >&2
+        echo "ERROR: mixed tabs and spaces: ${1}" >&2
```

Comment:
Does this syntax make the CI error window paint the error in red so it stands out?

The only line I see painted in red in an existing CI display is:
```
 ##[error]Bash exited with code '1'.
```
Maybe use that syntax?

See https://dev.azure.com/cncf/envoy/_build/results?buildId=79229&view=logs&j=c5dd2866-6ab3-5f3c-3a44-4cef0ec909b5&t=a9eb66d6-8944-5769-b3f7-476949dadcb8

Reply:
ill give it a whirl - it might be that console color codes are required if not - which is never pretty

Reply:
wow! that automagically worked, very handy!